Packs/Cloudflare_New/Integrations/CloudflareNew/CloudflareNew.py

```diff
@@ -0,0 +1,250 @@
+                ["action", "clientAsn", "clientCountryName", "clientIP", "clientRequestPath",
+                    "clientRequestQuery", "datetime", "source", "userAgent"],
+                removeNull=True,
+            )
+
+            # data_to_table = parse_list_to_table(act_result["cloudflare_events"]["zones"][0]["firewallEventsAdaptive"])
+            # return_results(
+            #     "Type": entryTypes["note"],
+            #     "Contents": data_to_table,
+            #     "ContentsFormat": formats["table"]
+            # )
+
+            return_results(
+                {
+                    "Type": entryTypes["note"],
+                    "ContentsFormat": formats["json"],
+                    "Contents": act_result,
+                    "ReadableContentsFormat": formats["markdown"],
+                    "HumanReadable": human_readable,
+                    "IgnoreAutoExtract": False
+                }
+            )
+
+            if not cloudflare_events['zones'][0]['firewallEventsAdaptive']:
+                continue
+            print("Zone_name: {}".format(zone_names[i]), zone_id)
+        else:
+            raise Exception(f"Error: {response.status_code}, {response.text}")
+            return_error(f"Error: {response.status_code}, {response.text}")
+
+    return logs
+
+
+def main():
+    apikey = demisto.params().get("API-key")
+    args = demisto.args()
+    # LOG(args)
+
+    zone_id = args.get("ZoneID")
+    zone_ids = argToList(args.get("zone_ids"))
+    start_time = args.get("start_time")
+    end_time = args.get("end_time")
+    source_ip = args.get("source_ip")
+
+    baseserver = (
+        demisto.params()["url"][:-1]
+        if (demisto.params()["url"] and demisto.params()["url"].endswith("/"))
+        else demisto.params()["url"]
+    )
+
+    verify_certificate = not demisto.params().get("insecure", False)
+    proxies = handle_proxy()
+
+    headers = {
+        "Authorization": f"Bearer {apikey}",
+        "Content-Type": "application/json",
+    }
+
+    command = demisto.command()
+    try:
+        client = Client(baseserver, verify_certificate, proxies)
+        commands = {
+            "get-firewall-events": get_firewall_events,
+            "get-zone-details": get_zone_details
+        }
+        if command == "test-module":
+            result = test_module(client, headers)
+            return_results(result)
+        elif command == "get-firewall-events":
+
+            result = get_firewall_events(
+                client, zone_ids=zone_ids, start_time=start_time, end_time=end_time, source_ip=source_ip, headers=headers
+            )
+            return_results(result)
+        elif command == "get-zone-ids":
+            result = get_zone_ids(client, headers)
+            return_results(result)
+        elif command == "get-zone-details":
+            result = get_zone_details(client, zone_id=zone_id, headers=headers)
+            return_results(result)
```

Comment:
all commands should have a prefix of "product-object-" please change the commands names